### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by Amr-Ibra**
- Update translation template `org.gnome.Shell.Extensions.GSConnect.pot`
- Update German translation

### CODE REVIEW

**Comment by andyholmes** [comment]
Thanks for the contribution, but it looks like we already have German setup with Crowdin. Would you mind submitting your changes via https://crowdin.com/project/gsconnect?

File: po/de.po [hunk 1]
```diff
@@ -580,27 +603,9 @@ msgstr "Technische Details"
...
-msgid_plural "%d Connected"
-msgstr[0] "%d verbunden"
-msgstr[1] "%d verbunden"
-
-#. TRANSLATORS: A menu option to deactivate the extension
-#: src/extension.js:377
-msgid "Turn Off"
-msgstr "Deaktivieren"
+#: src/extension.js:46
+msgid "Sync between your devices"
```
Comment (Amr-Ibra) [inline po/de.po:607]: I didn't find this new string in Crowdin.

Comment (andyholmes) [reply po/de.po:607]: I've updated the translations on crowdin, can you see it now?

Comment (Amr-Ibra) [reply po/de.po:607]: Yes, and they are translated now in Crowdin.

**Comment by ferdnyc** [comment]
@Amr-Ibra It looks like you've been active on Crowdin (where German translation is at 100%; noice!), and all of the changes I spot-checked here look like they've been submitted (or updated further) over there. Can this PR be closed?

**Comment by Amr-Ibra** [comment]
> Can this PR be closed?

Yes, it's not needed anymore.

**Pull request closed without merging**